### ISSUE TITLE
The Alpine e-mail client can't be used with NVDA

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#5358]
**Reported by jkinnunen on 2015-09-13 19:29**
Those who are using the Alpine e-mail client with JAWS would like to use it also with NVDA, but currently that's not possible due to the fact that the menus (e.g. the main menu where one can choose to compose a message or configure settings) can't be navigated with the arrow keys, nor can screen review be used to examine the application window.

Alpine can be downloaded from [here.](http://patches.freeiz.com/alpine/info/windows.html)

**Comment by jkinnunen** [comment]
Can something be done about this issue?

**Comment by jkinnunen** [comment]
Would it be possible to do something about this issue?

**Comment by Adriani90** [comment]
@jkinnunen can you still reproduce this issue with NVDA 2024.4 Beta and Alpine 2.26?
Did you already contact the alpine developers team or anyone who is working on Alpine? I suspect the accessibility is not implemented in the software itself at all and then NVDA cannot really do anything about it if they don't change anything on their end.

**Comment by XLTechie** [comment]
@Adriani90, this can definitely repro with Alpine 2.26 and modern NVDA. However, when using Screen Review mode, at least sometimes parts of Alpine can be accessed, including the main menu. However, navigation in this way, only works from bottom to top. As soon as you try to navigate down with review commands, the review location resets to the menu bar, which is quite undesirable.
Note, that in Alpine, the Main menu is not the same as any of the menus on the menu bar. The main menu is essentially the Alpine home screen, in modern terminology.

setting Alpine to use dialogs on its window config menu, and setting "show cursor" (search for it with the `w` command) under Alpine setup (`s`, `c`, from the main menu), can help a bit.

It is possible this could be made to work with an app module, but I have found object nav is not very useful as it stands. An extensive amount of work would have to be done, I think.

Using Alpine under WSL or ssh, is much easier.

**Comment by jkinnunen** [comment]
@Adriani90 This issue can be closed as Alpine is no longer important to me. However, if someone else would like to take this over, feel free to do so.

**Comment by XLTechie** [comment]
I have a particular fondness for Alpine, but I think the level of work to bring this support to core is not worth the number of users who might find it valuable. Closing as not of sufficient community interest.
